core/modules/user/user.install

```diff
@@ -169,6 +169,17 @@ function user_schema() {
   return $schema;
 }
 
+/**
+ * Implements hook_schema_alter().
+ */
+function user_schema_alter(&$schema) {
+  // Prevent fatal error if someone clears cache before running the update.
+  // @see https://github.com/backdrop/backdrop-issues/issues/5944#issuecomment-1416900861
+  if (backdrop_get_installed_schema_version('user') < 1025) {
+    unset($schema['users']['fields']['changed']);
```

Comment:
Would it make sense to also generate a watchdog warning?  There is no precedent of this type of preventive code in Backdrop when adding a table column, so I'm not sure.